.github/workflows/ci.yml

```
@@ -8,54 +8,52 @@ on:
-      uses: actions/setup-node@v2
-      with:
-        node-version: '14.x'
-
-    - name: Pre-commit check
-      run: prettier --check "./*.{js,json,md}"
\ No newline at end of file
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
```

actionlint (1.7.4)

the runner of "actions/setup-python@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)